modules/TrackTimingPileUpSubtractor.cc

```diff
@@ -74,7 +74,8 @@ void TrackTimingPileUpSubtractor::Init()
   fItVertexInputArray = fVertexInputArray->MakeIterator();
 
   // read resolution formula in m
-  fFormula->Compile(GetString("ZVertexResolution", "0.001"));
+  fZVertexResolution = GetDouble("ZVertexResolution", 3);
```

Comment:
I think this naming is confusing.
Shouldn't we use instead something like:

fNeutralMinSignificance
fChargedMinSignificance